docs/user/vulnerabilities/vulnerability/flags/flag/label-usage.en.md

```diff
@@ -1 +1,3 @@
 # Label - Usage
+
+Describes the most appropriate reason why the vulnerability is not exploitable in machine-readable form.
```

Comment:
Add:
If there is no machine-readable status justification available that fits the reason - the impact statement must be done through the `threats` element with the category `impact` and the description in `details`.